include/map.h

```diff
@@ -36,6 +36,61 @@ namespace grppi {
 @{
 */
 
+/**
+\brief Invoke \ref md_map on a data sequence.
+\tparam InputIterators Iterator types used for input sequences.
+\tparam InputIt Iterator type used for any input sequence.
+\tparam OtuputIt Iterator type used for the output sequence.
+\tparam Transformer Callable type for the transformation operation.
+\param ex Sequential execution policy object.
+\param firsts Tuple of Iterators to the first element in the inputs sequences.
+\param last Iterator to one past the end of one of the input sequence.
+\param first_out Iterator to first element of the output sequence.
+\param transform_op Transformation operation.
+\note The sequential_execution object acts exclusively as a tag type.
```

Comment:
????

Reply:
I suppose you refer to the `\note ` section.

I copy pasted from line 132. Should i remove it from there too?